### DISCUSSION TITLE
Answer in chat keeps repeating - infinite loop.

### DISCUSSION THREAD

**Discussion by dbgnvan2** [brianpetro/obsidian-smart-connections#684] [Q&A] [answered]
When I ask the "chat" about some thing I get an answer, but it keeps repeating and I have to quit Obsidian to get it to stop.  Any ideas. 

?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the difference between markdown and html?
system: Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).
user: what is the

**Comment by brianpetro** [comment]
Hi @dbgnvan2 

Which chat model are you using?

🌴

**Reply by dbgnvan2** [reply]
Smart Chat Model is Open Router  - -1=ai/yi-34jb.   
Notes Embedding Model - Nomic-embed-text -v1.5

**Reply by brianpetro** [reply]
I recommend sticking with the default BGE-micro-v2 embedding model, especially if you have any issues with the embedding process, like it going really slow.

Regarding the chat model, I haven't used that particular model. I recommend trying some others, Open Router has a lot of options, to see if the issue clears up.

**Reply by dbgnvan2** [reply, marked as answer]
thanks very much

**Reply by dbgnvan2** [reply]
All is working well now. thanks.
